cmd/world/main.go

```diff
@@ -1,9 +1,40 @@
 package main
 
 import (
+	"log"
+	"time"
+
+	"github.com/getsentry/sentry-go"
+
 	"pkg.world.dev/world-cli/cmd/world/root"
 )
 
 func main() {
+	// Sentry initialization
+	err := sentry.Init(sentry.ClientOptions{
+		Dsn:                "{input your DSN here}",
```

Comment:
This line breaks the tool (`sentry.Init: [Sentry] DsnParseError: invalid scheme`). Sentry should be disabled if a valid DNS isn't provided.

Reply:
updated